docs/coding-standards/tests/common-rule/mock-type-cast-prohibition.md

```
@@ -0,0 +1,30 @@
+# 4. モック型キャストの禁止
+
+### 規約
```

markdownlint-cli2 (0.20.0)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)